### DISCUSSION TITLE
Lightsail issue - upgrade instance issue

### DISCUSSION THREAD

**Discussion by rlowevoiply** [angristan/wireguard-install#481] [Help]
I have been running this setup in conjuncture with wg-portal and now have reached to max cpu/mem limits of this $10 a month aws box. I am moving to the $80 box but it seems that when i switch over the static ip address the wireguard connection doesn't work. i suspect it has something to do with the private ip of the lightsail box and ip tables maybe?

Any help would be apricated, ever other docker applicate seems to be working with out issue. Thanks for your time!

**Comment by x-FBi** [comment]
I'm happy to help.
Can you post any details?  Sometimes it's the DNS which isn't resolving.
